src/main/java/org/broadinstitute/hellbender/utils/pileup/ReadPileup.java

```diff
@@ -256,6 +272,13 @@ public int getNumberOfElements(final Predicate<PileupElement> peFilter){
     }
 
     /**
+     * Shortcut for get the number of deletions
+     */
+    public int getNumberOfDeletions() {
```

Comment:
I'm not a fan of such methods (trivial for clients to code themselves and begs to have a `getNumberOfElementsBeforeInsertion`, `getNumberOfElementsAfterDeletionEnd` etc - why not have one for every boolean method on `PileupElement`?). If you really want to keep it, add unit tests for it.